File: Learn/5. Stacks/reverseText.java
import java.util.Stack;

public class reverseText {
    public static void main(String[] args) {
        System.out.println(reverseString("12345"));
    }

    static String reverseString(String text) {
        Stack<Character> textStack = new Stack<>();
        String reversedText = "";
        for (int i = 0; i < text.length(); i++) {
            textStack.push(text.charAt(i));
        }
        while (textStack.isEmpty() != true) {
            reversedText += textStack.pop();
        }
        return reversedText;
    }

}

File: 5. Stacks/reverseText.js
let reverseText = (text) =>{
   let textStack = [];
   let reversedText= '';
   for(let i =0; i< text.length; i++){
        textStack.push(text[i]);
   }
   for(let i =0; i<text.length; i++){
       let character = textStack.pop();
       reversedText += character;
   }
   return reversedText;
}

console.log(reverseText('Aditya Negara'));